src/unix/process.c

```diff
@@ -424,20 +431,135 @@ int uv_spawn(uv_loop_t* loop,
 
   /* Acquire write lock to prevent opening new fds in worker threads */
   uv_rwlock_wrlock(&loop->cloexec_lock);
-  pid = fork();
 
-  if (pid == -1) {
-    err = UV__ERR(errno);
-    uv_rwlock_wrunlock(&loop->cloexec_lock);
-    uv__close(signal_pipe[0]);
-    uv__close(signal_pipe[1]);
-    goto error;
-  }
+#if defined(__APPLE__)
```

Comment:
It's probably worth adding a comment linking to https://github.com/libuv/libuv/issues/3050, with a brief explanation that we want to use `posix_spawn` instead of `fork/execve` on macOS >= 11.0

Reply:
👍